doc/source/installation/yum_repo.rst

```diff
@@ -18,6 +18,35 @@ Supported Releases:
 
 The *CentOS* repositories should work well with *Red Hat Enterprise Linux* too, provided that :program:`yum` is installed on the server.
 
+.. important::
+
+   *CentOS* 6 offers an outdated version of the ``curl`` library required by
+    |Percona Server|. The version of the ``curl`` library in *CentOS* 6, which
+    depends on the ``nss`` library, is known to create memory corruption
+    issues. This bug is `registered in Red Hat Bugzilla
+    <https://bugzilla.redhat.com/show_bug.cgi?id=1057388>`_. Its current status
+    is `CLOSED WONTFIX`.
+
+    To ensure that you use the latest version of the ``curl`` library with
```

Comment:
Add a note somewhere that this is not required if you do not use the keyring Vault plugin